test/evaluate.js

```diff
@@ -7,11 +7,24 @@ const { BigNumber } = ethers
 
 after(telemetry.close)
 
+// Get the details using this command:
+//   curl https://spark.fly.dev/rounds/520 | jq .
+const ROUND_WITH_KNOWN_DETAILS = 520
+
+const VALID_PARTICIPANT_ADDRESS = '0xf100Ac342b7DE48e5c89f7029624eE6c3Cde68aC'
+const VALID_MEASUREMENT = {
+  participantAddress: VALID_PARTICIPANT_ADDRESS,
+  cid: 'QmUuEoBdjC8D1PfWZCc7JCSK8nj7TV6HbXWDHYHzZHCVGS',
+  provider_address: '/dns4/production-ipfs-peer.pinata.cloud/tcp/3000/ws/p2p/Qma8ddFEQWEU8ijWvdxXm3nxU7oHsRtCykAaVz8WUYhiKn',
+  protocol: 'bitswap'
+}
+
 describe('evaluate', () => {
   it('evaluates measurements', async () => {
-    const rounds = { 0: [] }
+    const roundIndex = ROUND_WITH_KNOWN_DETAILS
```

Comment:
Discussion point: in the current proposal, all tests are fetching details of a real round from the SPARK API endpoint.

The upside is that we know that all parts fit together well (this is integration-level testing).

The downside is ~300-700ms overhead for every test.

The alternative is to inject the function for getting details of the current round, similarly to how we inject `ieContractWithSigner` and `web3Storage`.

@juliangruber WDYT?

Reply:
+1 to injecting the function, this will also give more confidence in the tests because a flaky service won't cause issues

Reply:
Done in [302d589](https://github.com/filecoin-station/spark-evaluate/pull/24/commits/302d589d3945aeeb08e5fdcae97b1ae0bcf14351)